intranet/static/js/eighth/schedule.js

```diff
@@ -37,110 +38,105 @@ $(function() {
 
         var capacitySum = 0;
         var val = $select.val();
-        if(val !== null) {
+        if (val !== null) {
             val.forEach(function(roomID) {
                 var c = all_rooms_lookup[roomID].capacity;
-                if (c == -1) c = Infinity;
+                if (c === -1) {
+                    c = Infinity;
+                }
                 capacitySum += c;
             });
         }
 
         if (capacitySum == Infinity) capacitySum = -1;
         var $capacityInput = $("input.capacity", $select.parent().parent());
-        var defaultActivityCapacity = $capacityInput.data("activity-default-capacity");
+        // var defaultActivityCapacity = $capacityInput.data("activity-default-capacity");
         var defaultCapacity = $capacityInput.data("default-capacity");
-        $capacityInput.attr("placeholder", capacitySum == 0 ? defaultCapacity : capacitySum);
+        $capacityInput.attr("placeholder", capacitySum === 0 ? defaultCapacity : capacitySum);
     });
 
     window.propagate_direction = "both";
     window.propagate_method = "only_new";
 
     window.updatePropagateDirection = function(opt) {
         window.propagate_direction = opt;
-        console.info("New propagate direction:", opt)
 
         var classes = ["fa-arrows-v", "fa-long-arrow-up", "fa-long-arrow-down"];
-        if(opt == "both") {
+        if (opt === "both") {
             $(".propagate > i").removeClass(classes[1]).removeClass(classes[2])
                                .addClass(classes[0]);
-        } else if(opt == "up") {
+        } else if (opt === "up") {
             $(".propagate > i").removeClass(classes[0]).removeClass(classes[2])
                                .addClass(classes[1]);
-        } else if(opt == "down") {
+        } else if (opt === "down") {
             $(".propagate > i").removeClass(classes[0]).removeClass(classes[1])
                                .addClass(classes[2]);
         }
     }
 
     window.updatePropagateMethod = function(opt) {
-        console.info("New propagate method:", opt);
         window.propagate_method = opt;
     }
 
     $(".propagate").click(function() {
         var field = $(this).attr("data-field");
         var input = $(this).attr("data-input");
         var el = $("#" + input);
-        console.debug("Propagate", field, input, el.length > 0);
 
         var rows = $(".schedule-activity-grid tr.form-row");
-        if(window.propagate_direction && window.propagate_direction == "down") {
-            var current_row = el.parent().parent();
-            var rows = [];
-            var row = current_row;
-            while(row && row.length > 0) {
+        var row;
+        var current_row;
+        if (window.propagate_direction && window.propagate_direction === "down") {
+            current_row = el.parent().parent();
+            rows = [];
+            row = current_row;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'current_row' is not in camel case.](https://www.codacy.com/app/tjcsl/ion/file/2642918672/issues/source?bid=3176160&fileBranchId=3299251#l92)